course/templates/course/_course_menu.html

```diff
@@ -14,7 +14,7 @@
 </li>
 
 {% if instance.language|first == "|" %}
-<li role="presentation" class="header">
+<li  class="header">
```

Comment:
You left extra whitespace in the `<li>` elements in this file. Do you mind cleaning it up?